tembo-cli/src/cli/context.rs

```diff
@@ -92,62 +91,43 @@ pub fn tembo_credentials_file_path() -> String {
 pub fn list_context() -> Result<Context, anyhow::Error> {
     let filename = tembo_context_file_path();
 
-    let contents = match fs::read_to_string(filename.clone()) {
-        std::result::Result::Ok(c) => c,
-        Err(e) => {
-            bail!("Error reading file {filename}: {e}")
-        }
-    };
+    let contents = fs::read_to_string(&filename)
+        .map_err(|err| anyhow!("Error reading file {filename}: {err}"))?;
 
-    let context: Context = match toml::from_str(&contents) {
-        std::result::Result::Ok(c) => c,
-        Err(e) => {
-            bail!("Issue with format of toml file {filename}: {e}")
-        }
-    };
+    let context: Context =
+        toml::from_str(&contents).map_err(|err| anyhow!("Error reading file {filename}: {err}"))?;
 
     Ok(context)
 }
 
 pub fn get_current_context() -> Result<Environment, anyhow::Error> {
     let context = list_context()?;
 
-    let profiles = list_credentail_profiles()?;
+    let profiles = list_credential_profiles()?;
 
-    for mut e in context.environment {
-        if e.set.is_some() && e.set.unwrap() {
```

Comment:
@shahadarsh The previous logic does not check the actual value of the `set` variable, is this intended?

Reply:
Yes, it only sets 1 value at a time so the value for others will be `None`.